packages/frontend/src/components/Tooltip.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import measureText from '../utils/measure-text'
+import './tooltip.css'
+import state from '../contexts/state'
+import { observer } from 'mobx-react-lite'
+
+
+export default observer(({
+  text,
+  maxWidth,
+  ...props
+}) => {
+  const ui = React.useContext(state)
```

Comment:
Make sure to deconstruct `ui` from the state store. This `ui` variable is actually the entire state